<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ToucheClavier">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Clavier Greque :</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>633</width>
       <height>128</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="touche_5">
        <property name="text">
         <string>ε</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="touche_12">
        <property name="text">
         <string>μ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="touche_19">
        <property name="text">
         <string>τ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="touche_18">
        <property name="text">
         <string>σ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="touche_17">
        <property name="text">
         <string>ρ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="touche_10">
        <property name="text">
         <string>κ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="touche_3">
        <property name="text">
         <string>γ</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="touche_24">
        <property name="text">
         <string>ω</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="touche_11">
        <property name="text">
         <string>λ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="touche_4">
        <property name="text">
         <string>δ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="touche_13">
        <property name="text">
         <string>ν</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="touche_6">
        <property name="text">
         <string>ζ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="touche_21">
        <property name="text">
         <string>φ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="touche_7">
        <property name="text">
         <string>η</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="touche_14">
        <property name="text">
         <string>ξ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="touche_9">
        <property name="text">
         <string>ι</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="touche_2">
        <property name="text">
         <string>β</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="touche_16">
        <property name="text">
         <string>π</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="touche_23">
        <property name="text">
         <string>ψ</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="touche_22">
        <property name="text">
         <string>χ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="touche_15">
        <property name="text">
         <string>ο</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="touche">
        <property name="text">
         <string>α</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="touche_8">
        <property name="text">
         <string>θ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="touche_20">
        <property name="text">
         <string>υ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupMot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Mot a trouver :</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupPendu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>pendu :</string>
    </property>
    <widget class="QLabel" name="mot_pioch">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>mot</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>tentative :</string>
     </property>
    </widget>
    <widget class="QLabel" name="tent">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="png">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>381</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_3">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>40</y>
       <width>511</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_4">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>30</y>
       <width>361</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/4.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/5.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/6.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>-50</y>
       <width>251</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/7.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>261</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/9.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/10.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>291</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/11.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="png_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../JeuGrque-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK__Release/8.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>lettres deja donnné :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lettres">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>541</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>lettre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Traduction : </string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_trad">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>traduc</string>
    </property>
   </widget>
   <widget class="QLabel" name="png_number">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>fichier</string>
    </property>
    <addaction name="actionNouveau_Mot"/>
    <addaction name="actionQuitter"/>
    <addaction name="actionAfficher_le_dictionaire"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau_Mot">
   <property name="text">
    <string>Nouveau Mot</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAfficher_le_dictionaire">
   <property name="text">
    <string>afficher le dictionaire</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
